### PR TITLE
small refactor to reduce complexity #hacktoberfest

### PR DESCRIPTION
**Pull request by codisart**
based on code climate analysis

### CODE REVIEW

File: src/Util/Filter.php [hunk 1]
```diff
@@ -17,18 +17,16 @@
  */
 final class Filter
 {
+    /** @var Blacklist */
+    private static $blacklist;
```
Comment (sebastianbergmann) [inline src/Util/Filter.php]: What is your reasoning for using static attributes here?

Comment (codisart) [reply src/Util/Filter.php]: I choose to add the attributes to avoid add many params on the private methods that I added.

Comment (sebastianbergmann) [reply src/Util/Filter.php]: But is there a specific reason why you declared these attributes `static`?

Comment (codisart) [reply src/Util/Filter.php]: It seems to me that we can not use non static attribut ion a static method.

https://3v4l.org/vqjCY

Comment (sebastianbergmann) [reply src/Util/Filter.php]: Fair point, I blocked out the fact that `Filter` still uses static methods.

Comment (sebastianbergmann) [reply src/Util/Filter.php]: I would prefer, though, not to have these attributes and that `$blacklist` and `$prefix` are passed as parameters to the private method. Thanks!

Comment (codisart) [reply src/Util/Filter.php]: ok I will do the change. 

Will you be interested to remove the static method from `Filter` in a future refactoring ?

**Pull request merged**